book/listings/list_widgets/6/main.rs

```diff
@@ -21,10 +21,7 @@ fn build_ui(app: &Application) {
     // ANCHOR: string_list
     // Create a `StringList` with number from 0 to 100_000
     // `StringList` implements FromIterator<String>
-    let model: StringList = (0..=100_000)
-        .into_iter()
-        .map(|number| number.to_string())
-        .collect();
+    let model: StringList = (0..=100_000).map(|number| number.to_string()).collect();
```

Comment:
```suggestion
    let model = (0..=100_000).map(ToString::to_string).collect::<StringList>();
```
maybe?

Reply:
Sure

Reply:
It didn't work, and fixing it made the code uglier.
So I am sticking with the original code :)